src/cpp/fastdds/xtypes/dynamic_types/DynamicTypeBuilderImpl.hpp

```diff
@@ -82,6 +85,9 @@ class DynamicTypeBuilderImpl : public traits<DynamicTypeBuilder>::base_type
 
     traits<DynamicType>::ref_type build() noexcept override;
 
+    ReturnCode_t copy_from(
+            traits<DynamicTypeImpl>::ref_type type);
+
 protected:
 
     traits<DynamicTypeBuilderImpl>::ref_type _this ();
```

Comment:
Style (maybe uncrustify is messing with this). This suggestion applies to every `_this()` method defined in this PR.

```suggestion
    traits<DynamicTypeBuilderImpl>::ref_type _this();
```

Reply:
Done in c8b1745ba

Reply:
This style error is still present in the public include headers. Please, fix it.

Reply:
Done in c8f6dde98